mmdet3d/core/post_processing/box3d_nms.py

```diff
@@ -134,3 +134,47 @@ def aligned_3d_nms(boxes, scores, classes, thresh):
 
     indices = boxes.new_tensor(pick, dtype=torch.long)
     return indices
+
+
+def aligned_bev_nms(boxes, scores, classes, thresh):
```

Comment:
Can we directly use the 2D NMS implemented in mmcv by passing only the 4 necessary dimensions?